server/index.js

```diff
@@ -65,4 +60,25 @@ app.waitReady = function (callback, interval = 100) {
   }, interval);
 };
 
+/**
+ * Verify that all required environment variables are defined
+ * Otherwise, the process is stopped
+ */
+function checkEnv() {
+  let needToExit = false;
+  const keys = ["ACCESS_TOKEN_KEY", "REFRESH_TOKEN_KEY"];
+  for (const key of keys) {
+    if (!process.env[key]) {
+      Logger.error(
+        new Error(`The ${key} environment variable is required but not defined in .env.`)
```

Comment:
Can we insert a link to the [docs](https://github.com/ValFraNath/guacamole/wiki/Production-deployment) here?

Reply:
yep !